NGitLab/Impl/RepositoryClient.cs

```diff
@@ -109,7 +109,11 @@ public IEnumerable<Commit> GetCommits(GetCommitsRequest request)
 
     public CompareResults Compare(CompareQuery query)
     {
-        return _api.Get().To<CompareResults>(_repoPath + $@"/compare?from={query.Source}&to={query.Target}");
+        var url = _repoPath + $"/compare?from={query.Source}&to={query.Target}";
```

Comment:
`CompareAsync` also needs to be adjusted. You should probably move to a new centralized method the construction of the URL, to avoid repeating code.